ansible/roles/shell/templates/bash_profile

```diff
@@ -12,7 +12,7 @@ fi
 
 # User specific environment and startup programs
 
-PATH=$PATH:$HOME/bin:~/.composer/vendor/bin
+PATH=$PATH:$HOME/bin:~/.composer/vendor/bin:/usr/local/bin
```

Comment:
I think you had issues when installing composer, if you revert this change destroy + rebuild you should not need it.

Please give this a go and let me know if your still having issues.